src/d3a/d3a_core/export.py

```diff
@@ -143,8 +148,6 @@ def area_tree_summary_to_json(self, data: Dict):
 
     def raw_data_to_json(self, time_slot, data: Dict):
         subdirectory = pathlib.Path(self.directory, "raw_data")
```

Comment:
I suggest to add this as a self member and also reuse it in the area_tree_summary_to_json (where you can also omit the check of existence) Thanks!